README.md

```diff
@@ -60,6 +60,51 @@ This option enables you to develop your custom MATLAB commands for running tests
 
   ![new_custom_script_example](https://user-images.githubusercontent.com/47204011/55625021-32904400-57c5-11e9-86b7-478b930796c0.png)
 
+## Configuring "Multi-configuration"(matrix) project.
+The Jenkins plugin for MATLAB can be used inside of "Multi-configuration" (matrix) projects. Matrix projects allow jobs to be repeated with different configurations, such as testing on multiple platforms or testing against multiple MATLAB versions.
+
+#### Matrix build for "Automatic" option.
+
+1) Create a "Multi-configuration" project.
+
+![image](https://user-images.githubusercontent.com/47204011/62458632-0e586a00-b79b-11e9-8611-3671adb8c289.png)
+
+2) Add User-defined axis for different MATLAB versions as shown below.
+
+![image](https://user-images.githubusercontent.com/47204011/62459358-c6d2dd80-b79c-11e9-90dc-37891b8179ea.png)
+
+3) Replace the user defined axis variable in MATLAB root appropriately.
+
+![image](https://user-images.githubusercontent.com/47204011/62459137-3c8a7980-b79c-11e9-9bee-305b4cabfd42.png)
+
+4) Save and run the build.
+
+#### Matrix build for "Custom" option.
+
+1) Create a "Multi-configuration" project.
+
+![image](https://user-images.githubusercontent.com/47204011/62458632-0e586a00-b79b-11e9-8611-3671adb8c289.png)
+
+2) Add User-defined axis for different MATLAB versions as shown below.
```

Comment:
@mw-akumar  : Word "user-defined" should be consistent across doc. at few places its written like "user defined" & at few places its "User-defined"

Reply:
Done. changed it to "User-defined" across the doc.